friend/models.py

```diff
@@ -0,0 +1,60 @@
+from pyexpat import model
+from django.db import models
+from django.conf import settings
+from django.dispatch import receiver
+from django.utils import timezone
+
+class FriendList(models.Model):
+    author = models.OneToOneField(settings.AUTH_USER_MODEL, on_delete=models.CASCADE, related_name="author")
+    friends = models.ManyToManyField(settings.AUTH_USER_MODEL, blank=True, related_name="friends")
```

Comment:
Won't we store redundant information then?
I.e. 
```js
{
  author: user1,
  friends: [user2]
},
{
  author: user2,
  friends: [user1]
}
```
Would it be better if we created a table that just does foreign keys
```js
{
  friend1: user1
  friend2: user2
}
```
taking advantage of the fact that true friendship is bidirectional?

Reply:
Then we can do something like
```py
        friend2s = Friend.objects.filter(friend1=self.bob)
        friend1s = Friend.objects.filter(friend2=self.bob)
        allFriends = []
        for friend1 in friend1s:
            allFriends.append(friend1.friend1)
        for friend2 in friend2s:
            allFriends.append(friend2.friend2)
```
to find all friends